tests/system/test_misc.c

```diff
@@ -322,3 +322,17 @@ int cmd_extra(int argc, char **argv)
 
    return rc;
 }
+
+int cmd_test_uids(void)
+{
+   int rc = 0;
+   rc |= syscall(SYS_getuid);
```

Comment:
This is not a competitive programming exercise :-)
Please use:
```
DEVSHELL_CMD_ASSERT(syscall(...) == 0);
```
It might look worse, but in reality is better because it will indicate exactly where the failure has happen. In tests, it matters a lot *how* we fail, not just that we either pass or fail.

Reply:
Perfect, i was actually wondering if there was a way to fail with more information. Turns out there is.